the_mines/get_html.py

```diff
@@ -0,0 +1,47 @@
+from argparse import ArgumentParser
+import requests
+import shutil
+
+
+def get_html():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--matchday", action="store", required=True, help="ex. 8",
+    )
+    parser.add_argument(
+        "--season", action="store", required=True, help="ex. 2019/2020",
+    )
+    args = parser.parse_args()
```

Comment:
These are all really good `args` to have.  Have you considered moving them out to before the method call (in `if __name__ == ...`)? That way we could have `matchday` and `season` as parameters.  Makes it easier to test different cases and we could call it from other modules with params.